resources/rinkeby-testnet/README.md

```diff
@@ -10,20 +10,18 @@ To use [the protocol](https://rinkeby.angle.money) in testnet, you need to conne
 
 ![Metamask Rinkeby](../../.gitbook/assets/metamask-rinkeby-userguide.jpg)
 
-On Angle testnet, you will need ETH to pay for gas, and wETH, wBTC, USDC or DAI to use the app. We built a faucet at [rinkeby.angle.money/\#/faucet](https://rinkeby.angle.money/#/faucet) so you can receive these tokens quickly!
+On Angle testnet, you will need ETH to pay for gas, and wETH, wBTC, USDC or DAI to use the app. We built a faucet at [faucet.angle.money](https://faucet.angle.money/) so you can receive these tokens quickly!
 
 Here are the steps to follow:
 
 1. Post a tweet mentionning [@AngleProtocol](https://twitter.com/AngleProtocol/) and containing your Ethereum address.
 
    ![Tweet Screenshot](../../.gitbook/assets/tweet-address-userguide.png)
 
-2. Go to [rinkeby.angle.money/\#/faucet](https://rinkeby.angle.money/#/faucet), paste the link of your tweet, and click on **Get Tokens**.
+2. Go to [faucet.angle.money](https://faucet.angle.money/), paste the link of your tweet, and click on **Use Faucet**.
 3. After a few minutes, you should receive some ETH, wETH, wBTC, USDC, and DAI.
```

Comment:
Put the addresses of the corresponding tokens (because there are many test DAI, USDC and so)
Also ANGLE are distributed here